Gemfile

```diff
@@ -66,6 +66,7 @@ group :development, :test do
   gem "factory_girl_rails"
   gem 'jasmine-rails'
   gem 'jasmine-jquery-rails'
+  gem 'pdf-reader', github: 'yob/pdf-reader'
```

Comment:
I should note, the reason I had to do this was I wanted Page#orientation from https://github.com/yob/pdf-reader/pull/95